app/views/user_responses/_allresponses.html.erb

```diff
@@ -11,36 +11,40 @@
         <% comments_detail << answer.detail %>
       <% end %>
     <% end %>
-    <% @survey.questions.each_with_index do |val , index| %>
-      <p><%= val.statement %></p>
-      <% comments_index = 0 %>
-      <% val.options.each_with_index do |option_value, option_index| %>
-        <% if val.question_type == 'Checkbox' %>
-          <% if selected_option_ids.include? option_value.id %>
-            <%= check_box_tag :option, 1, true, disabled: true %>
-          <% else %>
-            <%= check_box_tag :option, 1, false, disabled: true %>
-          <% end %>
-        <% elsif val.question_type == 'Radio Buttons' %>
-          <% if selected_option_ids.include? option_value.id %>
-            <%= radio_button_tag "option #{radio_button_id}", nil, true, class: '', disabled: true %>
-          <% else %>
-            <%= radio_button_tag "option #{radio_button_id}", nil, false, class: '', disabled: true %>
-          <% end %>
-          <% radio_button_id += 1 %>
-        <% elsif val.question_type == 'Comment Box'  %>
-          <%= text_field_tag :option, "#{comments_detail[comments_index]}", class: '', readonly: true %>
-          <% comments_index += 1 %>
-        <% elsif val.question_type == 'True / False'  %>
-          <% if selected_option_ids.include? option_value.id %>
-            <%= radio_button_tag "option #{radio_button_id}", nil, true, class: '', disabled: true %>
-          <% else %>
-            <%= radio_button_tag "option #{radio_button_id}", nil, false, class: '', disabled: true %>
+    <div class = "text-left survey-box container mt-3 shadow p-3 mb-5 bg-white rounded">
+      <% @survey.questions.each_with_index do |question , index| %>
+        <p><%= question.statement %></p>
+        <% comments_index = 0 %>
+        <% question.options.each_with_index do |option, option_index| %>
+          <% if question.question_type == 'Checkbox' %>
+            <% if selected_option_ids.include? option.id %>
+              <%= check_box_tag :option, 1, true, disabled: true %>
```

Comment:
true -> selected_option_ids.include? option.id 
no need of if else and two check_box_tags